src/v2/guide/computed.md

```diff
@@ -179,84 +179,79 @@ computed: {
 // ...
 ```
 
-Now when you run `vm.fullName = 'John Doe'`, the setter will be invoked and `vm.firstName` and `vm.lastName` will be updated accordingly.
+Bây giờ, khi bạn gán `vm.fullName = 'Phu Ba'`, thì setter sẽ được gọi, lúc này `vm.firstName`, `vm.lastName` sẽ được cập nhật lại dựa vào những thay đổi trong setter.
 
-## Watchers
+## Watcher
 
-While computed properties are more appropriate in most cases, there are times when a custom watcher is necessary. That's why Vue provides a more generic way to react to data changes through the `watch` option. This is most useful when you want to perform asynchronous or expensive operations in response to changing data.
+Thuộc tính computed thích hợp cho hầu hết các trường hợp, nhưng cũng có lúc cần tới những watcher tùy chỉnh. Đó là lí do tại sao Vue cung cấp một cách khai quát hơn để phản ứng (react) lại với việc thay đổi dữ liệu trong qua `watch`. Cách sử dụng này rất hữu ích khi bạn muốn thực hiện những tính toán không đồng bộ và tiếu tốn cho việc thay đổi dữ liệu.
```

Comment:
* Typo "khai quát"
* Typo "tiếu tốn." Anh nghĩ nên dịch là "tốn kém."
* Câu cuối dịch sai ý. "In response to" không thể dịch là "cho."